browser/ui/tabs/brave_tab_menu_model.cc

```diff
@@ -111,4 +119,57 @@ void BraveTabMenuModel::Build(int selected_tab_count) {
   InsertItemWithStringIdAt(close_other_tabs_index.value_or(GetItemCount()),
                            CommandCloseDuplicateTabs,
                            IDS_TAB_CXMENU_CLOSE_DUPLICATE_TABS);
+
+  if (base::FeatureList::IsEnabled(tabs::features::kBraveSplitView)) {
+    BuildItemsForSplitView(browser, tab_strip_model, indices);
+    return;
+  }
+}
+
+void BraveTabMenuModel::BuildItemsForSplitView(
+    Browser* browser,
+    TabStripModel* tab_strip_model,
+    const std::vector<int>& indices) {
+  CHECK(browser);
+
+  auto index = *GetIndexOfCommandId(TabStripModel::CommandReload);
```

Comment:
Maybe worth a note that we want to insert the `SplitView` commands after the reload command.